### PR TITLE
Onboarding azure-communication-messages SDK

### PR DESCRIPTION
**Pull request by arifibrahim4**
# Description
Onboarding Azure Communication Messages SDK.

The service is used to send messages to different channel. Currently we support only WhatsApp Channel. SDK provides feature to send message, view channel template list and download media.

Typespec => https://github.com/Azure/azure-rest-api-specs/pull/27405
API View => https://apiview.dev/Assemblies/Review/6562e5ee918f497cba32cbf009b4de95?revisionId=d890a68ed27d4a3092da420822fd08fb&doc=False

# All SDK Contribution checklist:
- [x] **The pull request does not introduce [breaking changes]**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: eng/versioning/version_client.txt [hunk 1]
```diff
@@ -74,6 +74,7 @@ com.azure:azure-communication-common-perf;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-communication-email;1.0.9;1.1.0-beta.1
 com.azure:azure-communication-identity;1.5.1;1.6.0-beta.1
 com.azure:azure-communication-jobrouter;1.1.0;1.1.1
+com.azure:azure-communication-messages;1.0.0
```
Comment (srnagar) [inline eng/versioning/version_client.txt:77]: This should have both current and released versions. 

```suggestion
com.azure:azure-communication-messages;1.0.0;1.0.0
```

Comment (arifibrahim4) [reply eng/versioning/version_client.txt]: fixed it. Thank you!

File: sdk/communication/azure-communication-messages/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+## 1.0.0-beta.1 (Unreleased)
+
+### Features Added
+
+### Breaking Changes
+
+### Bugs Fixed
+
+### Other Changes
```
Comment (srnagar) [inline sdk/communication/azure-communication-messages/CHANGELOG.md:21]: Remove this section
```suggestion
```

**Review by juancamilor — APPROVED** [review]

**Review by Shamkh — APPROVED** [review]
Looks good

**Comment by arifibrahim4** [comment]
@Azure/dpg-devs for awareness

**Comment by arifibrahim4** [comment]
@JimSuplizio , @alzimmermsft , @samvaity , @g2vinay , Pls provide your approval and let me know if you have any question. Thank you!

**Review by juancamilor — APPROVED** [review]

**Pull request merged**